influxdb3/tests/server/query.rs

```diff
@@ -884,11 +888,11 @@ async fn api_v1_query_json_format() {
                         "host",
                         "usage"
                       ],
-                      "name": "mem",
+                      "name": "cpu",
```

Comment:
does this imply the iox impl has a different default sorting? just curious; I don't think its a concern either way.